tests/parsers/agilent_gen5/testdata/fluorescence/endpoint_single_plate_example.json

```diff
@@ -0,0 +1,648 @@
+                                            "value": 504.0,
+                                            "unit": "nm"
+                                        }
+                                    }
+                                ]
+                            },
+                            "measurement identifier": "AGILENT_GEN5_TEST_ID_5",
+                            "sample document": {
+                                "sample identifier": "SPL18",
+                                "location identifier": "B3",
+                                "well plate identifier": "Plate 1"
+                            },
+                            "error aggregate document": {
+                                "error document": [
+                                    {
+                                        "error": "<0.000",
+                                        "error feature": "[Concentration]"
+                                    }
+                                ]
+                            },
+                            "compartment temperature": {
+                                "value": 21.6,
+                                "unit": "degC"
+                            },
+                            "fluorescence": {
+                                "value": 24.0,
+                                "unit": "RFU"
+                            }
+                        }
+                    ],
+                    "measurement time": "2024-08-15T11:51:56+00:00",
+                    "plate well count": {
+                        "value": 6,
+                        "unit": "#"
+                    },
+                    "analytical method identifier": "Q:\\prod\\neo\\neo2_1_24030606\\protocols\\QuantiFluorDNA.prt",
+                    "experimental data identifier": "\\\\CB-NAS01\\Data\\prod\\neo\\neo2_1_24030606\\protocols\\BenchlingConnectTest2.xpt",
+                    "container type": "well plate"
+                }
+            }
+        ],
+        "calculated data aggregate document": {
+            "calculated data document": [
+                {
+                    "calculated data name": "Blank 504,531",
+                    "calculated result": {
+                        "value": 112.0,
+                        "unit": "(unitless)"
+                    },
+                    "calculated data identifier": "AGILENT_GEN5_TEST_ID_6",
+                    "data source aggregate document": {
+                        "data source document": [
+                            {
+                                "data source identifier": "AGILENT_GEN5_TEST_ID_0",
+                                "data source feature": "fluorescence"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "calculated data name": "[Concentration]",
+                    "calculated result": {
+                        "value": -0.0,
```

Comment:
@james-leinas Should we still create a calculated data document for a value that was already reported in the error document? (`AGILENT_GEN5_TEST_ID_0` has an error document for the `[Concentration]` feature).

Reply:
@joshua-benchling, to @slopez-b point, we should disregard creating a calculated data document for values reported in the error document. Since the calculated data aggregate/calculated data document isn't required in the schema.